react/src/SearchVariants/components/SearchVariantsPage.tsx

```diff
@@ -26,6 +37,19 @@ const useStyles = makeStyles(theme => ({
         paddingLeft: theme.spacing(1),
         paddingRight: theme.spacing(1),
     },
+    wrapper: {
+        margin: theme.spacing(1),
+        minWidth: 166,
+        justifyContent: "space-between",
+        position: "relative",
+        display: "flex",
+        alignItems: "center",
+    },
+    buttonProgress: {
+        color: green[500],
+        position: "absolute",
+        marginLeft: 50,
+    },
```

Comment:
The download button appears to have a wide space from the side margin. 

![image](https://user-images.githubusercontent.com/11793063/126506730-d728f397-d614-4821-8d66-8a57c7ade5ce.png)

I figured out how to fix it while maintaining the circular progress overlay, based on https://material-ui.com/components/progress/#interactive-integration.

```suggestion
    wrapper: {
        margin: theme.spacing(1, 0.5, 1, 1),
        // minWidth: 166,
        justifyContent: "space-between",
        position: "relative",
        display: "flex",
        alignItems: "center",
    },
    buttonProgress: {
        color: green[500],
        position: "absolute",
        // marginLeft: 50,
        top: "50%",
        left: "50%",
        marginLeft: -12,
        marginTop: -12,
    },
```